docs/development/workflow/development_workflow.rst

```diff
@@ -422,6 +422,13 @@ it into Astropy:
    pull request message.  This is still a good way to start a preliminary
    code review.
 
+You may also opt to open a work-in-progress pull request.
```

Comment:
Thanks, reads well!  Though I think I'd merge it with the sentence above (and at the same indentation), as in:
```
   If you don't think your request is ready to be merged, just say so in your
   pull request message, or opt to open a work-in-progress pull request (by
   clicking on the small down arrow next to the button and selecting "Create
   draft pull request"). Either way works to start preliminary code review.

   In addition, if your commits...
```

Reply:
Please see second commit. Thanks for the review!